qa/restart-upgrade/src/test/java/org/opensearch/knn/bwc/ModelIT.java

```diff
@@ -140,13 +143,14 @@ public void testKNNModelDefault() throws Exception {
     }
 
     // KNN Delete Model test for model in Training State
+    @Ignore
     public void testDeleteTrainingModel() throws Exception {
-        byte[] testModelBlob = "hello".getBytes(StandardCharsets.UTF_8);
-        ModelMetadata testModelMetadata = getModelMetadata();
-        testModelMetadata.setState(ModelState.TRAINING);
         if (isRunningAgainstOldCluster()) {
+            byte[] testModelBlob = "hello".getBytes(StandardCharsets.UTF_8);
+            ModelMetadata testModelMetadata = getModelMetadata();
+            testModelMetadata.setState(ModelState.TRAINING);
             addModelToSystemIndex(TEST_MODEL_ID_TRAINING, testModelMetadata, testModelBlob);
-        } else {
+
```

Comment:
Could you tell why this change is needed? Especially on why else statement is not required?

Reply:
The previous test relied on the bug where models stuck in training state would still be in training state when the cluster restarts.  Since I modified that, the test is no longer working.

Reply:
I am still don't' understand why we need a test for only old cluster. Anyway, this test need to be rewritten so we can stop the further discussion of this matter.

Reply:
I agree, it should probably be a normal integration test instead of bwc